wordpress-mobile/WordPress-Android
# AppBar refresh - Reader and Notifications

This PR updates AppBar design and adds lift on scroll functionality to the following screens (and related nested screens):

- Reader
- Notifications

Main Reader and Notification screens accessible from the bottom nav are already updated, so no need to check them.

To test:

For all screens:
- Make sure nothing looks out of place (including ActionMode) :)
- Make sure the toolbar state is restored on configuration change.


Test 1:

Navigate to the Reader.
Check the following screens, and make make sure that lift on scroll works:
- Search (there is a pre-existing issue with the whole AppBAr being focused alongside the search input field).
- Blog's post list.
- Post details (+swipe right/left in ViewPager)
- Post Comments
- Post "Liked By" screen.


Test 2:

Navigate to the Notifications.
- Check various notification detail pages and make sure that lift on scroll works  (+swipe right/left in ViewPager).
- Check Notification settings.

PR submission checklist:
